src/Fixer/Whitespace/ArrayIndentationFixer.php

```diff
@@ -300,7 +300,7 @@ private function getLineIndentation(Tokens $tokens, $index)
 
     private function extractIndent($content)
     {
-        if (Preg::match('/\R([\t ]*)[^\r\n]*$/', $content, $matches)) {
+        if (Preg::match('/\R([\t ]*)[^\r\n]*$/D', $content, $matches)) {
```

Comment:
let me merge this anyway, as we have nice utests for the fixer, but...
i'm not big fan of non-PCRE modifiers